sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -1031,7 +1065,18 @@ object Radar {
                             return
                         }
 
-                        verificationManager.getIntegrityToken(config.googlePlayProjectNumber, config.nonce) { integrityToken, integrityException ->
+                        if (config.nonce == null) {
+                            handler.post {
+                                callback?.onComplete(RadarStatus.ERROR_UNKNOWN)
+                            }
+
+                            return
+                        }
+
+                        val stringToHash = verificationManager.getRequestHash(location, config.nonce);
+                        val requestHash = hashSHA256(stringToHash)
```

Comment:
nit: move the `hashSHA256(stringToHash)` into the `getRequestHash` function? since we do this after in both places.  then can just be `val requestHash = verificationManager.getRequestHash(...)`